packages/test/test-end-to-end-tests/src/test/agentScheduler.spec.ts

```diff
@@ -29,8 +28,8 @@ const tests = (argsFactory: () => ITestObjectProvider) => {
 
         beforeEach(async () => {
             const container = await args.makeTestContainer();
-            scheduler = await requestFluidObject<TaskManager>(container, taskSchedulerId)
-                .then((taskManager) => taskManager.IAgentScheduler);
+            scheduler = await requestFluidObject<IAgentScheduler>(container, taskSchedulerId)
+                .then((agentScheduler) => agentScheduler.IAgentScheduler);
```

Comment:
same as above, seems redundant

Reply:
It's redundant on current version, but on N-1 the request returns the `TaskManager`, so this step ensures that on both N and N-1 we're really dealing with the `AgentScheduler`

Reply:
Can this be removed in a future version when backcompat is no longer a problem? If so please add a comment.

Reply:
Comment added in latest